recipes/datamountaineer-schemaregistry/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: datamountaineer-schemaregistry
+  version: "0.2"
+
+source:
+  git_url: https://github.com/datamountaineer/python-serializers.git
+  git_rev: 3a55967d29524f406548f2afa844f92f18d12a7d
```

Comment:
There is no link :wink: 

You just create patches with the changes and add them here. I know that using a specific commit is easier, but the patch is easier to follow and understand.